conf/external.yaml

```diff
@@ -92,6 +89,9 @@ clustering_descriptor_selector:
 # and model state.
 runner:
   sequential:
+    run_time: 10
+    run_time_unit: "MINUTES"
```

Comment:
@fruch did you intend to shorten it from 2 hours to 10 minutes?

Reply:
no, I copy pasted for a different place, anyhow I think it's an example configuration.
But I'll fix that

Reply:
fixed